pipeline-modules/deployment-service/azure/azure-functions.yaml

```diff
@@ -1,15 +1,15 @@
 provisioner: azure
-name: azure-functions-code
+name: azure-functions-deploy
```

Comment:
why is rename required?
Currently renaming a module creates two copies of it. the old module will not be deleted.
if not important, keep the same name.